tests/p_rsync/0-install-rsync.sh

```diff
@@ -3,14 +3,14 @@
 
 t_Log "Running $0 - installing rsync and xinetd."
 
-if [ "$centos_ver" = "8" ] ; then
-  t_InstallPackage xinetd rsync rsync-daemon
+if [ "$centos_ver" = "8" ] || [ "$centos_ver" = "9" ] ; then
+  t_InstallPackage rsync rsync-daemon
```

Comment:
`-ge 8` test here. Is xinetd not needed?

Reply:
No. Systemd is sufficient.